dev/archery/archery/integration/datagen.py

```diff
@@ -1659,6 +1734,13 @@ def _temp_path():
         .skip_category('Java')  # TODO(ARROW-7779)
         .skip_category('JS'),
 
+        generate_rle_case()
```

Comment:
s/rle/ree

Reply:
this is what i get for re-using changes lol. this time i'll do a full search/replace *sigh*